Lib/test/libregrtest/runtest.py

```diff
@@ -206,6 +206,8 @@ def _test_module(the_module):
 def _runtest_inner2(ns, test_name):
     # Load the test function, run the test function, handle huntrleaks
     # and findleaks to detect leaks
+    minruns = sys._getopcacheminruns()
+    sys._setopcacheminruns(1)
```

Comment:
That's wrong. Only dash_R() should be modified in libregrtest/refleak.py. You must not disable the opcache in runtest.py.

Reply:
Sorry, this was supposed to reset the original value. Check out my last fixup (bd3a951ff81eab28b77d05290a2466573c41eda2),

Reply:
I think you misunderstood my comment. You must not modify this function but dash_R() function.